vllm/config/compilation.py

```diff
@@ -524,13 +541,16 @@ def __post_init__(self) -> None:
                 )
 
     def init_backend(self, vllm_config: "VllmConfig") -> Union[str, Callable]:
-        if self.level == CompilationLevel.NO_COMPILATION:
+        if self.level == CompilationMode.NO_COMPILATION:
```

Comment:
Can you make the codebase use `mode` only and never use level? That'll make it easier to deprecate.